apps/web/src/app/[lang]/network/page.tsx

```
@@ -19,14 +30,31 @@ export default async function Index(props: PageProps) {
   return (
     <Container>
       <Text className="text-9 text-center flex mb-8">
         Explore hundreds of Spaces in the Hypha Network
       </Text>
       <SpaceSearch />
-      <SpaceGroupSlider spaces={spaces} type="Hypha" getHref={getHref} />
+      {uniqueCategories.map((category) => (
+        <SpaceGroupSlider
+          key={category}
+          spaces={spaces.filter(
+            (space) => space.categories && space.categories.includes(category),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)